plugins/snippets/nvim-snippets.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
```

```suggestion
}:
let
  inherit (lib) types;
  inherit (lib.nixvim) defaultNullOpts;
in
helpers.neovim-plugin.mkNeovimPlugin config {
```

This is a brand-new coding style that we are trying out.